Vagrantfile

```diff
@@ -163,6 +167,23 @@ def suse_common(config)
     java_package: "java-1_8_0-openjdk-devel")
 end
 
+def sles_common(config)
```

Comment:
makes sense, did that... not sure if this is the syntactically right approach, feel free to correct